### ISSUE TITLE
Testing HomoDepth on own data

### ISSUE THREAD

**Issue by hanqiuli** [UCI-ISA-Lab/MultiHeadDepth-HomoDepth#4]
Hi there! Thanks for the amazing work and making all of it public.

I am trying to see if I can use HomoDepth for real-time depth estimation.
 
To run the pre-trained DTU checkpoint, aside from resizing the input images into the same one as training, is there anything else one should take into account?

**Comment by Yongfan-Liu** [comment]
Hello @hanqiuli ,

1. HomoDepth is designed for misaligned input stereo images. If your data is rectified, please consider MultiHeadDepth.
2.  Note that the baseline of the DTU is 6~18 cm. If the baseline of your data is not in this range, the accuracy may not be good.
3. If your dataset contains the ground truth of depth map and homography information of images (or others like intrinsic, extrinsic, or project matrix), it would be better to fine-tune the model based on your dataset.
